KIPs/kip-19.md

```diff
@@ -0,0 +1,125 @@
+
+#### New Configurations
+
+| Subject    |      Actors      |                                                            Permissions | Proposed Contract      |
+| ---------- | :--------------: | ---------------------------------------------------------------------: | ---------------------- |
+| KNC Token  |      Miner       |                                                               Mint KNC | Long Timelock Executor |
+|            |   Proxy Admin    |                                              Change KNC implementation | Long Timelock Executor |
+|            | Withdrawal Admin | Withdraw tokens from the contract due to accidental transfer of tokens | 3/5 Multisig           |
+| Governance |      Admin       |                     Authorize and deauthorize executors and strategies | Long Timelock Executor |
+|            |  Executor Admin  |                       Queue, cancel, and execute proposal transactions | Governance Contract    |
+|            |   DAO Operator   |                                            Create and cancel proposals | 5/9 Multisig           |
+
+#### Timelock Executor Configurations
+
+| Properties         |                                                          Definition                                                          | Short Timelock | Long Timelock |
+| ------------------ | :--------------------------------------------------------------------------------------------------------------------------: | -------------: | ------------- |
+| Delay              |                                              Waiting time to execute proposals                                               |       12 hours | 1 week        |
+| Grace period       |                                         Proposal execution period before it expires                                          |         4 days | 7 days        |
+| Min delay          |                                         Minimum time in the range of a delay config                                          |         1 hour | 24 hours      |
+| Max delay          |                                         Maximum time in the range of a delay config                                          |       24 hours | 2 weeks       |
+| Min vote duration  |                                            Minimum voting duration of a proposal                                             |         4 days | 7 days        |
+| Max voting options |                                           Maximum number of options in a proposal                                            |              8 | 8             |
+| Vote differential  | The difference needed between the votes behind the For/Against options (percentage of Total KNC supply in the market needed) |          0.01% | 15%           |
+| Min quorum         |                      Minimum percentage of Total KNC supply in the market needed for the winning option                      |             4% | 20%           |
+
```

Comment:
Can add here: The new Timelock Executor has been deployed here: 0x7d4d05B1a1E5775a9C6ca248ABBE629B52C1D9D9 with corresponding configurations